content/vendor/guides/_index.md

```diff
@@ -54,8 +55,19 @@ In this guide, we'll explore how you can leverage the wealth of software bundled
 
 ###### [Manage Customer Entitlements](/vendor/guides/entitlements)
 
+-->
+
+
 ## Advanced
 
+#### [Airgapped Existing Cluster](/vendor/guides/airgapped-existing-cluster)
+
+This guide covers an end-to-end example of deploying to an airgapped existing cluster with KOTS.
+It walks through all the steps to create an environment in GCP with an airgapped Kubernetes cluster, Docker registry, and workstation from which to do the deployment, plus a jump box / bastion for accessing the airgapped instances.
```

Comment:
I stumbled a bit while reading the above sentence. I wonder if the following is a bit more clear. I made an attempt...
```suggestion
It walks through all the steps to create an airgapped environment in GCP, including the Kubernetes cluster setup, Docker Registry configuration, local workstation setup, and a jump box / bastion for accessing the airgapped instances.
```